plugins/janus_videoroom.c

```diff
@@ -1671,7 +1541,47 @@ static void janus_videoroom_message_free(janus_videoroom_message *msg) {
 	g_free(msg);
 }
 
+void janus_videoroom_codecstr(janus_videoroom * videoroom, char* audio_codecs, char* video_codecs, int str_len, const char* split)
```

Comment:
Just nitpicking here, please bear with me! Per code style, for pointers the convention is `type *variable`, which is not matched in the signature above. I noticed the same thing in a few other places as well, so please fix those too.